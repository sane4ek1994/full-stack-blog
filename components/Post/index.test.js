import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Post } from './index'

it('expect to render Post component', () => {
  const mockPost = {
    title: 'Additional post',
    description: 'text',
    image: { asset: { _ref: 'image-17019027e1c135d42a7d79d9529b6463856eb408-119x124-svg' } },
    slug: { _type: 'slug', current: 'additional-post' }
  }
  const postComponent = shallow(<Post {...mockPost} />)

  expect(toJson(postComponent)).toMatchSnapshot()
})
