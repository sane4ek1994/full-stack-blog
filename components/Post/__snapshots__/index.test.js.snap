// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Post component 1`] = `
<ForwardRef(LinkComponent)
  className="post"
  href="/post/additional-post"
>
  <a
    className="postLink"
  >
    <Title
      className="postTitle"
      type="small"
    >
      Additional post
    </Title>
    <div
      className="postContent"
    >
      <div>
        <Image
          height="100"
          src="https://cdn.sanity.io/images/xu6emnmj/production/17019027e1c135d42a7d79d9529b6463856eb408-119x124.svg"
          width="100"
        />
      </div>
      <p
        className="postDescription"
      >
        text
      </p>
    </div>
  </a>
</ForwardRef(LinkComponent)>
`;
